src/utils/request.ts

```diff
@@ -149,3 +149,20 @@ export function getRequestURL(
   const path = getRequestPath(event);
   return new URL(path, `${protocol}://${host}`);
 }
+
+export function getRequestIP(
+  event: H3Event,
+  opts: { xForwardedFor?: boolean } = {},
+) {
+  const nonProxyIp = (event.node.req.connection as any).remoteAddress;
```

Comment:
We shall support (and prefer + type) `event.context.clientAddress` to support integration with nodeless runtimes.

Reply:
Good point, can you see if I set this up correctly.